.swiftpm/xcode/xcshareddata/xcschemes/swift-composable-architecture.xcscheme

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Huh, what's with the scheme?

Reply:
deleted, think it's cause I opened the `Package.swift` directly instead of the workspace.